.gitignore

```diff
@@ -18,3 +18,5 @@ WebOb.egg-info/
 pytest*.xml
 coverage*.xml
 .pytest_cache/
+.idea/
```

Comment:
This should go in your user .gitignore, not here.

Reply:
Thanks, I fixed the relative commit.